### DISCUSSION TITLE
AttributeError: module 'onnxruntime' has no attribute 'InferenceSession'

### DISCUSSION THREAD

**Discussion by With-Darkness** [microsoft/onnxruntime#21948] [Release Q&A]
Hi, All the members of Github.
It's a great opportunity to use onnxruntime package as a new AI developer.
I have some problems with using onnxruntime package, that is the error: AttributeError: module 'onnxruntime' has no attribute 'InferenceSession'.
Initially, I used onnxruntime-gpu v 1.16.3 and It worked well, there is no issue like AttributeError: module 'onnxruntime' has no attribute 'InferenceSession'.
But I upgraded the onnxruntime-gpu from 1.16.3 to 1.18.1 and occurred an error.
I can't figure out this issue. Doesn't the onnxruntime package upper 1.16.3  support InferenceSession?
If it is right, what is the other way instead of InferenceSession?
I would appreciate your help as soon as possible.

**Comment by RyanUnderhill** [comment]
Can you show the smallest example of what you're doing that causes the error? It sounds like it could be similar to this issue: https://github.com/microsoft/onnxruntime/issues/8623

Specifically, try this: Running `help(rt)` after `import onnxruntime as rt` to see if it's what you expect it to be.
